<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="L-SFM38WDJC2" name="TripAction" tests="3" failures="1" timestamp="30 Dec 2018 18:28:11 GMT" time="25.473" errors="0">
  <testcase name="searchHotels" time="25.443" classname="tripAction.testcases.Bookings"/>
  <testcase name="testAssertion1" time="0.003" classname="tripAction.testcases.Bookings"/>
  <testcase name="testAssertion2" time="0.003" classname="tripAction.testcases.Bookings">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at tripAction.testcases.Bookings.testAssertion2(Bookings.java:96)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray2(ReflectionUtils.java:208)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:159)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:87)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:95)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testAssertion2 -->
</testsuite> <!-- TripAction -->
